test/e2e/e2e_test.go

```diff
@@ -562,9 +562,8 @@ func (p *options) DefaultOpenStackOptions() hypershiftopenstack.RawCreateOptions
 		OpenStackExternalNetworkID: p.configurableClusterOptions.OpenStackExternalNetworkID,
 		NodePoolOpts: &openstacknodepool.RawOpenStackPlatformCreateOptions{
 			OpenStackPlatformOptions: &openstacknodepool.OpenStackPlatformOptions{
-				Flavor:         p.configurableClusterOptions.OpenStackNodeFlavor,
-				ImageName:      p.configurableClusterOptions.OpenStackNodeImageName,
-				AvailabityZone: p.configurableClusterOptions.OpenStackNodeAvailabilityZone,
```

Comment:
we don't want all Nodepools to have AZ specified, so we remove it from here. This change is expected.